spellbook/models/nft/nft_schema.yml

```diff
@@ -14,21 +14,99 @@ models:
       - &blockchain
         name: blockchain
         description: "Blockchain"
-      - name: project
+      - &project
+        name: project
         description: "Project"
```

Comment:
personally, I think if the description is the same as the column name, we can just omit. unless we are adding some detail, it's just duplicative.

Reply:
Agreed